File: Patterns/1. Square Patterns/04. Square Decreasing Numbers/sdn.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=n; i>=1; i--){
        for(int j=n; j>=1; j--){
            cout<<j<<' ';
        }
        cout<<'\n';
    }
}

/*
Output

5 4 3 2 1 
5 4 3 2 1 
5 4 3 2 1 
5 4 3 2 1 
5 4 3 2 1 

*/

File: Patterns/01. Square Patterns/04. Square Decreasing Numbers/sdn.java
import java.util.*;
public class sdn{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=n; i>=1; i--){
            for(int j=n; j>=1; j--){
                System.out.print(j + " ");
            }
            System.out.println();
        }

        sc.close();
    }
}

/*
Output

5 4 3 2 1 
5 4 3 2 1 
5 4 3 2 1 
5 4 3 2 1 
5 4 3 2 1 

*/